pipeline_dp/dp_computations.py

```diff
@@ -435,3 +436,30 @@ def compute_dp_var(count: int, normalized_sum: float,
         dp_mean += compute_middle(dp_params.min_value, dp_params.max_value)
 
     return dp_count, dp_mean * dp_count, dp_mean, dp_var
+
+
+def compute_dp_count_noise_params(
+        dp_params: MeanVarParams) -> Tuple[str, float]:
+    """Computes noise type and standard deviation for DP count.
+
+    Args:
+        dp_params: parameters for DP count aggregation.
+
+    Returns:
+        Tuple (noise type ('laplace' or 'gaussian'), noise std).
+    """
+    l0_sensitivity = dp_params.l0_sensitivity()
```

Comment:
I don't like that this logic lives in PipelineDp. It feels to me Building Blocks Library would be the ideal place for it. All this computation is already done there, no?

Reply:
Somewhere in `Count` algorithm, but it's not used in PipelineDP.

And that sensitivity logic is that simple, that we shouldn't hide it inside the library.  And the using the library for this makes things more complicated :).

Reply:
Sure, it's simple but I think it'd be better to not duplicate code as much as possible. I'd vote for being mindful of complexity of maintaining all places with DP logic

Reply:
Duplicating code is only one dimension of simplicity of maintenance :) and there are always tradeoff :). 

For example here:  
1. Creating `Count`/`BoundedSum` objects are slow in Python (only ~30-50K/sec).
2. `Count` object has not very friendly API for getting the amount of noise :)
3. PyDP `Count` wrapper doesn't support Gaussian noise

And independently of Count. PipelineDP has vector aggregation, we need to compute sensitivities/amount of noise for it, and it's more complicated then in scalar case, it would be weird when simple logic is outsourced to the library, while complex is in PipelineDP :)

Reply:
IMO, problems 2 & 3 (and 4, meaining your last point about complexity and vector aggregations) are things that can be solved. I understand that the way C++ library and by extension Python library are implemented are not very compatible with  what we'd like to do here, but I do think there is a lot of benefit to migrating as much DP logic as we can to the libraries if we can do it in a nice and seamless way.

But we can discuss that later on, it's not specific to this PR :)

Reply:
Sure, It would be great to have all logic in one place :). Let's discuss later :)